javascript/spliffy/package.json

```
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@srfnstack/spliffy": "^0.7.0"
```

please use `~` to stick to the same minor version

```suggestion
    "@srfnstack/spliffy": "~0.7.0"
```

---

Updated. Thanks for checking.